drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/columns/control/AttributeColumnConfigRow.java

```diff
@@ -40,80 +39,78 @@ public AttributeColumnConfigRow() {
-                    @Override
-                    public void onDefaultValueChanged(DefaultValueWidgetFactory.DefaultValueChangedEvent event) {
-                        final AttributeCol52 editedColumn = originalColumn.cloneColumn();
-                        editedColumn.setDefaultValue( event.getEditedDefaultValue() );
-                        presenter.getActiveDecisionTable().updateColumn( originalColumn,
-                                                                         editedColumn );
-                    }
-                });
+                             presenter.isActiveDecisionTableEditable(),
+                             new DefaultValueWidgetFactory.DefaultValueChangedEventHandler() {
+                                 @Override
+                                 public void onDefaultValueChanged(DefaultValueWidgetFactory.DefaultValueChangedEvent event) {
+                                     final AttributeCol52 editedColumn = originalColumn.cloneColumn();
+                                     editedColumn.setDefaultValue(event.getEditedDefaultValue());
+                                     presenter.getActiveDecisionTable().updateColumn(originalColumn,
+                                                                                     editedColumn);
+                                 }
+                             });
+
+        hideColumnCheckBox = view.addHideColumnCheckBox(attributeColumn,
+                                                        new ClickHandler() {
+                                                            @Override
+                                                            public void onClick(ClickEvent event) {
+                                                                final AttributeCol52 editedColumn = originalColumn.cloneColumn();
+                                                                editedColumn.setHideColumn(hideColumnCheckBox.getValue());
+                                                                presenter.getActiveDecisionTable().updateColumn(originalColumn,
+                                                                                                                editedColumn);
+                                                            }
+                                                        });
 
-        view.add( new HTML( "&nbsp;&nbsp;" ) );
-        hideColumnCheckBox = view.addHideColumnCheckBox(attributeColumn, new ClickHandler() {
-            @Override
-            public void onClick(ClickEvent event) {
-                final AttributeCol52 editedColumn = originalColumn.cloneColumn();
-                editedColumn.setHideColumn( hideColumnCheckBox.getValue() );
-                presenter.getActiveDecisionTable().updateColumn( originalColumn,
-                                                                 editedColumn );
-            }
-        });
+        if (presenter.isActiveDecisionTableEditable()) {
```

Comment:
Other methods for adding widget into the view takes as argument `isEditable` and just enable or disable the added widget. I think it could be fine to add argument `isEditable` into `addRemoveAttributeButton` to keep adding widgets into view more conssistent. Let me know what do you think.